website/src/docs/hotchocolate/api-reference/migrate-from-10-to-11.md

```diff
@@ -460,6 +460,24 @@ New
         .AddType<FooType>();
 ```
 
+## IgnoreField
+
+The order of the parameters in ignore field has changed. The schema name is now optional and was moved to the end.
+
+Old
+
+```csharp
+    services.AddStitchedSchema(x => x.IgnoreField("SchemaName", "TypeName, "FieldName"));
+```
+
+New
+
+```csharp
+    services
+        .AddGraphQLServer()
+        .IgnoreField("TypeName, "FieldName", "SchemaName")
```

Comment:
this is a mean one :D but optionals belong to the back of the chain.

Reply:
yup it is... :D